examples/oauth/src/components/Auth.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import { IframeStamper } from "@turnkey/iframe-stamper";
```

Comment:
any particular reason this example is using `iframe-stamper` directly instead of using `sdk-react`?

Reply:
consistency with all of our existing authentication examples - happy to make a second effort to move things from iframe-stamper + http -> sdk-browser + sdk-react.

Reply:
depends on bandwidth, if you're in a crunch i'm cool w merging this as is. gotta update all the other ones in bulk at some point!

Reply:
Pretty straight forward change - may as well include here - nw :)